platforms/documentation/docs/src/samples/readme-templates/common-body.adoc.template

```diff
@@ -1,4 +1,4 @@
-NOTE: You can open this sample in an <<gradle_ides.adoc#gradle_ides,IDE that supports Gradle>>.
+NOTE: You can open this sample in an link:{userManualPath}/gradle_ides.html#gradle_ides[IDE that supports Gradle].
```

Comment:
💅 Those repeated lines look like something that should be generated and/or included from the snippet (instead of duplication). Not to be addressed by this PR, though.

Reply:
The samples section is going to get a major overhaul next year.